extension/src/ui/components/VideoDataSyncDialog.tsx

```diff
@@ -14,6 +13,7 @@ import Toolbar from '@material-ui/core/Toolbar';
 import Typography from '@material-ui/core/Typography';
 import makeStyles from '@material-ui/styles/makeStyles';
 import Switch from '@material-ui/core/Switch';
+import { LabelWithHoverEffect } from '@project/common/components/SettingsForm';
```

Comment:
As mentioned above, import `LabelWithHoverEffect` from its own file rather than pulling in `SettingsForm` here.